src/components/ellipsis/ellipsis.tsx

```diff
@@ -12,13 +16,15 @@ export type EllipsisProps = {
   rows?: number
   expandText?: string
   collapseText?: string
+  stopPropagation?: PropagationEvent[]
```

Comment:
我在想这个属性是不是改个名字比较好，`stopPropagation` 的话，很容易让人认为是整体的事件冒泡阻止（对于目前其他组件来说都是对整体的事件冒泡阻止）

```suggestion
  stopPropagationForExpandButtons?: PropagationEvent[]
```

Reply:
改了下，卡神再看看